### DISCUSSION TITLE
Security Vulnerability faced in nginx Container Images

### DISCUSSION THREAD

**Discussion by VijayRajVg** [nginx/kubernetes-ingress#6322] [Q&A]
Please find the list of CIB ID’s below:
CON-Image-CVE-2024-25062
CON-Image-CVE-2024-25062
CON-Image-CVE-2024-25062
CON-Image-CVE-2024-25062

**Comment by AlexFenlon** [comment]
Hi @VijayRajVg ,

Just a few questions to ask;
Can you tell me what project you are using?
Cam you tell me what version of NIC this is running against?
Can you tell me what image you are running this on?
Can you also tell me what tool you are using to get this information?
